plugins/node/opentelemetry-instrumentation-aws-lambda/src/instrumentation.ts

```diff
@@ -77,6 +78,18 @@ export class AwsLambdaInstrumentation extends InstrumentationBase {
 
   constructor(protected override _config: AwsLambdaInstrumentationConfig = {}) {
     super('@opentelemetry/instrumentation-aws-lambda', VERSION, _config);
```

Comment:
WDYT about modifying the config object before using it as a paramater in the `super` call? might be a bit more robust IMO

Reply:
7f40ea85